lib/model/rwfolder.go

```diff
@@ -194,13 +172,11 @@ func (f *sendReceiveFolder) Serve() {
 			curSeq, ok := f.model.RemoteSequence(f.folderID)
 			if !ok || curSeq == prevSec {
 				l.Debugln(f, "skip (curSeq == prevSeq)", prevSec, ok)
-				f.pullTimer.Reset(f.sleep)
 				continue
 			}
 
 			if err := f.model.CheckFolderHealth(f.folderID); err != nil {
 				l.Infoln("Skipping pull of", f.Description(), "due to folder error:", err)
-				f.pullTimer.Reset(f.sleep)
```

Comment:
What's going to trigger this now, given the timer is not doing it anymore?

Reply:
A call to SchedulePull, i.e. an index update, a scan or a change in cluster config. The point of this PR is to not pull every 10s but only if something changed.

Reply:
Sure, but let's say we have an outdated folder that was unmounted, with inotify or something like that that has watches setup for an unmounted folder. We mount the folder, the inotify is somewhat useless, never triggers a scan (as fd's point at the wrong directory), how would we know a directory is alive again?

Reply:
If a folder is stopped, it gets started -> scanned. If a folder is in state error, this error can only be removed during a scan.

Reply:
Right, ok.